src/Helldivers-2-Core/Mapping/V1/AssignmentMapper.cs

```diff
@@ -44,6 +44,7 @@ private Assignment MapToV1(Dictionary<string, Models.ArrowHead.Assignment> trans
         var titles = translations.Select(assignment => new KeyValuePair<string, string>(assignment.Key, assignment.Value.Setting.OverrideTitle));
         var briefings = translations.Select(assignment => new KeyValuePair<string, string>(assignment.Key, assignment.Value.Setting.OverrideBrief));
         var descriptions = translations.Select(assignment => new KeyValuePair<string, string>(assignment.Key, assignment.Value.Setting.TaskDescription));
+        var expiration = translations.Select(assignment => DateTime.UtcNow.AddSeconds(assignment.Value.ExpiresIn)).FirstOrDefault();
```

Comment:
While LINQ executes lazily and will only execute this once, it reads at first glance as if this will execute for all translations.

A few lines above we declare an `invariant` variable for when we want to fetch information that isn't translation dependent (like the expiration).
So basically:
```suggestion
        var expiration = DateTime.UtcNow.AddSeconds(invariant.ExpiresIn);
```